etc/module.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="MageOS_CommonAsyncEvents"/>
```

Comment:
Sequence for other Magento modules is missing

Reply:
Added.